### DISCUSSION TITLE
ECE%20MEng%40Cornell/

### DISCUSSION THREAD

**Discussion by giscus** [opencsapp/opencsapp.github.io#207] [Announcements]
# ECE%20MEng%40Cornell/

None

https://opencs.app/ECE%20MEng%40Cornell/

**Comment by BigBrotherJu** [comment]
> coop 对于 22fall 来说已经不存在了

来源是哪里啊？

**Comment by Hancen-Fan** [comment]
博主可以更新一下了，cornell tech也是这副德行，令人无语。本人psu cs(3.98) -> wisc cs(3.85)。被翠拒。官网上写的study area都算挺软的，但招的基本都是ece的学生。。。
